src/platform/site-wide/sass/modules/_m-crisis-line.scss

```diff
@@ -1,3 +1,7 @@
+div#modal-crisisline {
```

Comment:
should we also assign a lower z-index to `nebula_div_btn`?
Doing so using the devtools achieved the following on staging:
<img width="1369" alt="screen shot 2018-09-24 at 2 56 14 pm" src="https://user-images.githubusercontent.com/16051172/45981484-0754bd00-c00a-11e8-92ed-330507d4d404.png">

Reply:
Good idea we'll go with that